stixpy/coordinates/tests/test_transforms.py

```diff
@@ -62,7 +62,7 @@ def test_stx_to_hpc_obstime_end():
     stix_coord_rt = hp.transform_to(STIXImaging(obstime=times[0], obstime_end=times[-1]))
 
     stix_coord_rt_interp = hp.transform_to(STIXImaging(obstime=times[1]))  # noqa: F841
-    assert_quantity_allclose(0 * u.deg, stix_coord.separation(stix_coord_rt), atol=1e-17 * u.deg)
+    assert_quantity_allclose(0 * u.deg, stix_coord.separation(stix_coord_rt), atol=1e-9 * u.deg)
```

Comment:
what happened here?

Reply:
Not sure.  But a billionth of a degree is still well within any tolerance we would believe...